scripts/cd-dev.sh

```
@@ -0,0 +1,100 @@
+echo "$APPLICATION_SECRET" > $APP_DIR/config/application-secret.yml
+
+# 보안을 위해 권한 설정 (선택사항)
+chmod 600 $APP_DIR/config/application-*.yml
+echo "✅ 환경 설정 파일 생성 완료"
+
+# [4] Docker Hub 로그인
+echo ""
+echo "[2] Docker Hub 로그인"
+echo "$DOCKERHUB_TOKEN" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
+
+if [ $? -eq 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)